### PR TITLE
react/week1

### PR DESCRIPTION
**Pull request by islam-fawzy25**

### CODE REVIEW

**Review by nuha90 — COMMENTED** [review]
good islam!

File: react/week1/TodoList/src/dash.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <thead>
+        <tr>
+          <th>ToDo</th>
+          <th>Description</th>
+          <th>Deadline Date</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.todoData.map((obj, index) => {
+          return <TableBody key={index} obj={obj} />;
```
Comment (nuha90) [inline react/week1/TodoList/src/dash.js:39]: you can use id in your array as key instead of index

File: react/week1/TodoList/src/dash.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <tr>
+      <td>{props.obj.todoItem} </td>
+      <td>{props.obj.description}</td>
+      <td>{props.obj.deadlineDate}</td>
+    </tr>
+  );
+}
+
+export default function TodoTable(props) {
+  console.log(props.todoData);
```
Comment (nuha90) [inline react/week1/TodoList/src/dash.js:27]: always push your code without console :)

**Comment by islam-fawzy25** [comment]
Thanks, @nuha90  it was really helpful comments

**Pull request merged**